rllib/examples/connectors/connector_v2_frame_stacking.py

```diff
@@ -86,27 +90,14 @@ def _make_learner_connector(input_observation_space, input_action_space):
     # We would like our frame stacking connector to do this job.
     tune.register_env(
         "env",
-        (
-            lambda cfg: (
-                EpisodicLifeEnv(  # each life is one episode
-                    MaxAndSkipEnv(  # frameskip=4 and take max over these 4 frames
-                        NoopResetEnv(  # perform n noops after a reset
-                            # partial(FrameStack, k=4)(  # <- no env-based framestacking
-                            NormalizedImageEnv(
-                                partial(WarpFrame, dim=64)(  # grayscale + resize
-                                    partial(
-                                        gym.wrappers.TimeLimit, max_episode_steps=108000
-                                    )(
-                                        gym.make(
-                                            "ALE/Pong-v5",
-                                            **dict(cfg, **{"render_mode": "rgb_array"})
-                                        )
-                                    )
-                                )
-                            )
-                        )
-                    )
-                )
+        lambda cfg: (
+            wrap_atari_for_new_api_stack(
```

Comment:
IsnÄt there a gym.Wrapper for framestacking?

Reply:
There is, but this would blow up the memory taken by the observations sometimes by n-frames.
This is just an example. I could actually show that using two connectors here (instead of such an env wrapper): learner AND env-to-module actually speeds up Atari runs by 5% :)